src/getCoinCombination.test.js

```diff
@@ -1,10 +1,42 @@
-'use strict';
+"use strict";
 
-describe('getCoinCombination', () => {
-  const { getCoinCombination } = require('./getCoinCombination');
+describe("getCoinCombination", () => {
+  const { getCoinCombination } = require("./getCoinCombination");
+  describe("getCoinCombination", () => {
```

Comment:
The use of a nested describe block with the same name 'getCoinCombination' is redundant. Consider removing the nested describe block to make the tests more readable.